src/main.rs

```diff
@@ -89,174 +64,12 @@ async fn main() -> error::Result<()> {
         tenant_store = Arc::new(tenant_database);
     }
 
-    let mut state = state::new_state(
+    let state = echo_server::state::new_state(
```

Comment:
Why is this now scoped?

Reply:
Was following compiler recommendations and don't see a problem with it?

Reply:
No problems, just wondered if you had a specific reason why :)